src/styles/blocks/calendar/calendar.scss

```diff
@@ -0,0 +1,43 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0 auto;
+  gap: $box-gap;
+  width: $calendar-width;
+  padding: $box-padding;
+
+  @each $day, $number in $days {
+    &--start-day--#{$day} :first-child {
+      margin-left: $number * ($box-size + $box-gap);
+    }
+  }
+
+  @for $day from 28 through 31 {
```

Comment:
if you chosen this way to hide blocks, add this numbers to variables

`$min-days-in-month: 28`
`$max-day-in-month: 31`